.changeset/smart-cows-tan.md

```diff
@@ -0,0 +1,9 @@
+---
+'@finos/legend-application-studio': patch
+'@finos/legend-extension-dsl-diagram': patch
+'@finos/legend-extension-store-service-store': patch
+'@finos/legend-graph': patch
+'@finos/legend-query-builder': patch
+---
+
+Minor improvements: Refactor code when creating an instance of Multiplicity ([#1542](https://github.com/finos/legend-studio/issues/1542)).
```

Comment:
don't think this minor refactor warrants a changeset. let leave this empty

Reply:
cleaned